### PR TITLE
Package bson2.0.0.1

### PR DESCRIPTION
**Pull request by nosman**
### `bson2.0.0.1`
Bson format encoding/decoding for Ocaml

---
* Homepage: https://github.com/nosman/Bson2
* Source repo: git+https://github.com/nosman/Bson2.git
* Bug tracker: https://github.com/nosman/Bson2/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by nosman** [comment]
In the `3 compilers` task, the build does: `install core v0.11.3    [required by bson2]`

However, my `bson2.opam` file specifies `"core" {>= "0.12.0"}`.

I'm not sure why this is happening, any ideas?

File: packages/bson2/bson2.0.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+maintainer: "stephanos.tsoucas@gmail.com"
+authors: "Stephanos Tsoucas"
+license: "Apache-2.0"
+homepage: "https://github.com/nosman/Bson2"
+bug-reports: "https://github.com/nosman/Bson2/issues"
+depends: [
+  "ocaml" {>= "4.04"}
+  "ounit" {with-test}
+  "dune" {build}
+  "core" {>= "0.12.0"}
```
Comment (kit-ty-kate) [inline packages/bson2/bson2.0.0.1/opam]:
```suggestion
  "core" {>= "v0.12.0"}
```
The `v` prefix is not optional

File: packages/bson2/bson2.0.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+synopsis: "Bson format encoding/decoding for Ocaml"
+maintainer: "stephanos.tsoucas@gmail.com"
+authors: "Stephanos Tsoucas"
+license: "Apache-2.0"
+homepage: "https://github.com/nosman/Bson2"
+bug-reports: "https://github.com/nosman/Bson2/issues"
+depends: [
+  "ocaml" {>= "4.04"}
+  "ounit" {with-test}
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/bson2/bson2.0.0.1/opam]:
```suggestion
  "dune" {>= "1.11"}
```
Dune shouldn't be tagged as a build dependency (see #14266) and the `dune-project` file in your repository indicates your project requires at least dune 1.11

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**